### PR TITLE
Enable Tide + Prow + Branch Protection on the community repo.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: prow/cluster/jobs/istio/community/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+approvers:
```
Comment (fejta) [inline prow/cluster/jobs/istio/community/OWNERS:1]: Is the approve plugin turned on in this repo?

**Review by fejta — APPROVED** [review]
/hold

File: prow/cluster/jobs/istio/community/OWNERS (hunk 1 above)
Comment (geeknoid) [reply prow/cluster/jobs/istio/community/OWNERS:1]: You mean on  test-infra?

I'm just  following the pattern of the other jobs, they  have a similar OWNERS file.

**Comment by geeknoid** [comment]
What's the hold for?

Thanks.

File: prow/cluster/jobs/istio/community/OWNERS (hunk 1 above)
Comment (fejta) [reply prow/cluster/jobs/istio/community/OWNERS:1]: oh right, I'm dumb

**Comment by fejta** [comment]
`/hold` == give you the chance to response to optional feedback by either addressing and/or unholding.

/hold cancel

**Pull request merged**